src/components/FormField/FormFieldSelect.js

```diff
@@ -1,36 +1,74 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Select from 'react-select';
+import { useField } from 'formik';
 import styles from './FormField.module.scss';
+import mainStyles from '~/assets/styles/main';
+import FormFieldErrorBoundary from '../error-boundaries/FormFieldErrorBoundary';
+import variables from '~/assets/styles/variables';
+
+const mapValues = (values) => (values ? values.map((val) => ({ label: val, value: val })) : []);
```

Comment:
maybe `getSelectValues` will be better for this function?

Reply:
Yeah, I renamed it as `getSelectedValuesAsObjects`